projects/dotcms-field-elements/src/components/dot-form/utils/fields.tsx

```diff
@@ -120,6 +120,16 @@ export const DotFormFields = {
 
     Tag: (field: DotCMSContentTypeField) => (
         <dot-tags
+            data={async (): Promise<string[]> => {
+                return fetch('//localhost:8080/api/v1/tags', {
```

Comment:
For some reason inside the `dot-form` this gets trigger everytime we add a tag.

Reply:
this 'localhost:8080' should not be hard code